components/table/src/simple-columns/simple-column-base.ts

```diff
@@ -68,6 +67,7 @@ export type DtSimpleColumnFormatFunction = (
   displayValue: any, // tslint:disable-line:no-any
 ) => string | DtFormattedValue;
 
+@Directive()
```

Comment:
Does the selectorless directive work in this case, or will this handled by their sub-classes?

Reply:
an abstract class that extends from a directive has to have the decorator according to the new Angular 9 this is correct.